drivers/adc/Kconfig.ltc2451

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Brill Power
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ADC_LTC2451
+	bool "LTC2451 driver"
+	default n
+
+if ADC_LTC2451
+
+config ADC_LTC2451_INIT_PRIORITY
```

Comment:
Is there a reason not to use `ADC_INIT_PRIORITY`?

Reply:
There wasn't a reason, has been updated!